.github/workflows/test.yml

```diff
@@ -7,6 +7,7 @@ on:
       - master
   schedule:
     - cron: '0 0 * * *'
+  workflow_dispatch:
```

Comment:
is this suppose to be empty or have something after the : ?

Reply:
This it is the syntax yes: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
You can specify additional options after the colon.
Maybe I can actually remove the colon itself. I will try

Reply:
The YAML is actually invalid without the colon.